DOCS-1680: Notice that the transaction id = order id within the notification url (#983)
MultiSafepay/docs: content/faq/api/how-does-the-notification-url-work.md

When the status of this transaction changes, we will notify your web server with this URL through a **GET** request.
https://yourdomain.com/index/paymentprovidernotification?invoice_id=840&transactionid=12345&timestamp=140292929

> __Please note:__ The *transaction_id* within the notification url should retain the same value as that of the *order_id*

Reviewer: ```suggestion
__Please note:__ Within the notification url, the *transaction_id* should have the same value as the *order_id*
```
